crates/conformance-test-utils/src/madara.rs

```diff
@@ -0,0 +1,154 @@
+use eyre::Result;
+use kakarot_rpc_core::client::constants::STARKNET_NATIVE_TOKEN;
+use reth_primitives::{U128, U256};
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_storage_var_address;
+
+use crate::types::{ContractAddress, StorageKey, StorageValue};
+
+/// Generates the genesis storage tuple for setting the storage of a Starknet contract.
+///
+/// This function calculates the storage key for the storage variable `storage_variable_name` and
+/// its keys. The resulting tuple represents the initial storage of the contract, where the storage
+/// key at a given `storage_key_offset` is set to the specified `storage_value`.
+pub fn genesis_set_storage_starknet_contract(
+    starknet_address: FieldElement,
+    storage_variable_name: &str,
+    keys: &[FieldElement],
+    storage_value: FieldElement,
+    storage_key_offset: u64,
```

Comment:
the `key` here is confusing wrt to the `keys` above, the former being the key of the storage and the latter the offset in the value. `storage_offset` is imho enough. For example, a storage is like

```
@storage_var
func balance(felt) -> Uint256
```

the `storage_key_value` is indeed for `Uint256.low` and `Uint256.high` iiuc

Reply:
gotcha, `storage_offset` is less confusing ✅  thanks!